scripts/pylint_extensions.py

```diff
@@ -162,10 +182,12 @@ def process_module(self, node):
         Args:
             node: astroid.scoped_nodes.Function. Node to access module content.
         """
-        file_content = node.stream().readlines()
+        file_content = [
+            utils.convert_to_unicode(line) for line in
```

Comment:
Same here? Maybe we should have a general method for accessing the content/names of files?

Reply:
done